src/Onboarding/Processor/BasicOnboardingProcessor.php

```diff
@@ -76,7 +76,7 @@ public function process(
             'partner_attribution_id' => $response['partner_attribution_id'],
         ]);
 
-        $permissionsGranted = (bool) $request->query->get('permissionsGranted', true);
+        $permissionsGranted = $request->query->get('permissionsGranted') === null ? true : (bool) $request->query->get('permissionsGranted');
```

Comment:
This looks strange, the previous implementation should work fine (we have `true` as a default value). Psalm couldn't handle it?

Reply:
in newer versions second parameter must be null || string so thats the way around it :/